src/netlify/index.ts

```diff
@@ -30,21 +26,20 @@ const connectionCache = new KeyedResolvOnce<bs.Connection>();
 export const connect: ConnectFunction = (
   db: Database,
   remoteDbName = "",
-  url = "http://localhost:8888?protocol=ws"
+  url = "netlify://localhost:8888?protocol=ws"
```

Comment:
same question as with aws

Reply:
same answer -> including that we should rename that from URL to -> connectUrl

and if it's a function than name it a function if it needs to be bound than =>